Sources/Vehicle/Templates/CarPlayServerListTemplate.swift

```diff
@@ -0,0 +1,82 @@
+import CarPlay
+import Foundation
+import HAKit
+import Shared
+
+@available(iOS 16.0, *)
+final class CarPlayServersListTemplate: CarPlayTemplateProvider {
+    private(set) static var carPlayPreferredServerKey = "carPlay-server"
+
+    var template: CPListTemplate
+    weak var interfaceController: CPInterfaceController?
+
+    private var preferredServerId: String {
+        prefs.string(forKey: CarPlayServersListTemplate.carPlayPreferredServerKey) ?? ""
+    }
+
+    init() {
+        self.template = CPListTemplate(title: "", sections: [])
+        template.tabTitle = L10n.Carplay.Labels.servers
+        template.tabImage = MaterialDesignIcons.cogIcon.carPlayIcon(color: nil)
+    }
+
+    func templateWillDisappear(template: CPTemplate) {
+        Current.servers.remove(observer: self)
```

Comment:
this isn't checking the template, is that an issue?

Reply:
Yup, I'll fix